exercise/staff_views.py

```diff
@@ -329,6 +329,8 @@ def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
         # submissions for this exercise
         submissions = Submission.objects.filter(exercise=self.exercise)
 
+        submitters_with_grading = submissions.exclude(grader=None).values_list('submitters__id', flat=True)
+
         # filter only the latest submissions
         submissions = (submissions.annotate(latest=Max('submission_time'))
```

Comment:
I don't think the `annotate`call for the `latest` submissions is working correctly. It doesn't seem to find the latest submission for each submitter separately.